src/test/java/Homework17.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+
+
+    @Test
+    public static void addSongToPlaylist () throws InterruptedException {
+
+        navigateToPage("https://bbb.testpro.io/");
+
+       provideEmail("shalinibaronia@gmail.com");
+       providePassword("te$t$tudent");
+       clickSubmit();
+        Thread.sleep(2000);
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)